src/base/lib/Convex/Utxos.hs

```diff
@@ -115,244 +111,97 @@ type AddressCredential = Shelley.PaymentCredential StandardCrypto
 
 {-| A set of unspent transaction outputs
 -}
-newtype UtxoSet ctx a = UtxoSet{ _utxos :: Map C.TxIn (C.InAnyCardanoEra (C.TxOut ctx), a) }
+newtype UtxoSet ctx era a = UtxoSet{ _utxos :: Map C.TxIn (C.TxOut ctx era, a) }
```

Comment:
Not sure the impact of this change in our other repo, but the advantage of using `InAnyCardanoEra` is that we can have utxos from multiple eras inside that type. Right now, you are restricted to a single era.

Reply:
When is this ever used / needed?